apps/web/app/actions.ts

```
@@ -1,119 +1,176 @@
 'use server'
 
+import { AuthError } from '@supabase/supabase-js'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)